packages/medusa/src/api/routes/admin/invites/index.ts

```diff
@@ -15,18 +16,18 @@ export const unauthenticatedInviteRoutes = (app) => {
 export default (app) => {
   app.use("/invites", route)
 
-  route.get("/", middlewares.wrap(require("./list-invites").default))
+  route.get("/", buildAuthenticatedHandler(middlewares.wrap(require("./list-invites").default)))
 
-  route.post("/", middlewares.wrap(require("./create-invite").default))
+  route.post("/", buildAuthenticatedHandler(middlewares.wrap(require("./create-invite").default)))
```

Comment:
Would it not suffice to just add: `route.use(middlewares.authenticate())`

Tested it locally, and seemed to be working fine. Any other particular reason for the helper, that I might have missed?

Reply:
Are you sure that if you had it using route.use it will not be applied to the unauthenticated routes as well ?
The reason for the helper is just to centralise the comment and that people does not search for it since it is a quick fix and it will need a better handling later on

Reply:
My bad you completely right, Ill move it to route.use I thought it was merged sorry :)